sdk/app_cpu1/common/drv/amds.h

```diff
@@ -63,6 +63,13 @@ typedef enum {
     AMDS_NUM_CHANNELS,
 } amds_channel_e;
 
+typedef enum {
+    AMDS_LOW_VOLTAGE_CARD = 0,
+    AMDS_HIGH_VOLTAGE_CARD,
+    AMDS_CURRENT_CARD_REVB,
+    AMDS_CURRENT_CARD_REVC
+} amds_card_t;
```

Comment:
```suggestion
typedef enum {
    AMDS_LOW_VOLTAGE_CARD = 0,
    AMDS_HIGH_VOLTAGE_CARD,
    AMDS_CURRENT_CARD_REVB,
    AMDS_CURRENT_CARD
} amds_card_t;
```

Since REVC hardware is the latest design, and presumably its method of implementing current measurement is what will be done going forward, let's make this the "default" by not having a `REVC` designator on the end of it.

The idea being here that we only call out past revisions by rev letter if they are not compatible with the current release. If we make a `REVD` it will presumably also have the same gain/offset. If it doesn't, then we would update our enum to look like this:

```C
typedef enum {
    AMDS_LOW_VOLTAGE_CARD = 0,
    AMDS_HIGH_VOLTAGE_CARD,
    AMDS_CURRENT_CARD_REVB,
    AMDS_CURRENT_CARD_REVC,
    AMDS_CURRENT_CARD
} amds_card_t;
```

When this gets merged, we should create an issue to update docs.amdc.dev and explain this.